src/index.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+
 import reportWebVitals from './reportWebVitals';
+import Calculator from './components/calculator';
+import './css/calculator.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```

Comment:
- [ ] The Calculator component should be imported into the App.js (main) component after that the App component will be imported into the index.js file. Please 🙏🏽 refactor your code to delete all the boilerplate that got created by the CRA (texts, images, and styles) in the App.js file for organized code.

Reply:
Thank you for pointing that out

Reply:
@franclobo Can you please complete the review on the dashboard? It is still showing "review pending".
Thanks